plugins/postcss-media-minmax/test/browser/basic.html

```diff
@@ -0,0 +1,1216 @@
+						@media (resolution > 191dpi) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution >= 192dpi',
+					style: `
+						@media (resolution >= 192dpi) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution = 192dpi',
+					style: `
+						@media (resolution = 192dpi) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution < 193dpi',
+					style: `
+						@media (resolution < 193dpi) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution <= 192dpi',
+					style: `
+						@media (resolution <= 192dpi) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+
+				{
+					title: 'resolution > 1dppx',
+					style: `
+						@media (resolution > 1dppx) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution >= 2dppx',
+					style: `
+						@media (resolution >= 2dppx) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution = 2dppx',
+					style: `
+						@media (resolution = 2dppx) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution < 3dppx',
+					style: `
+						@media (resolution < 3dppx) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+				{
+					title: 'resolution <= 2dppx',
+					style: `
+						@media (resolution <= 2dppx) {
+							target {
+								color: green;
+							}
+						}
+					`,
+				},
+
+			];
+
+			for (let testCase of testCases) {
+				// Native
+				{
+					const styleElement = document.createElement('style');
+					styleElement.textContent = testCase['style'];
+					document.head.append(styleElement);
+
+					var targets = document.querySelectorAll('target');
+					for (let target of targets) {
+						assert_equals(
+							window.getComputedStyle(target).color,
+							'rgb(0, 128, 0)',
+							testCase['title'] + ", target '" + target.classList[0] + "'"
+						);
+					}
+
+					styleElement.remove();
+				}
+
+				// Fallback
+				{
+					const styleElement = document.createElement('style');
+
+					const testCaseStyle = await (fetch(`/test/styles.css`, {
+						method: 'POST',
+						body: testCase['style'],
+					}).then((response) => response.text()));
+
+					if (testCaseStyle === testCase['style']) {
+						throw new Error(`Fallback failed for "${testCase['title']}"`);
+					}
+
+					styleElement.textContent = testCaseStyle;
+					document.head.append(styleElement);
+
+					var targets = document.querySelectorAll('target');
+					for (let target of targets) {
+						assert_equals(
+							window.getComputedStyle(target).color,
+							'rgb(0, 128, 0)',
+							testCase['title'] + ", target '" + target.classList[0] + "'"
+						);
+					}
+
+					styleElement.remove();
+				}
+			}
```

Comment:
Because we are behind browsers we can easily compare native and polyfill behavior. We might have false positives (test passes but there is also a bug) but if tests fail in the fallback and pass in native we are absolutely certain that there is a real bug.